task3.py

```diff
@@ -0,0 +1,66 @@
+"""
+Задание 3
+Дана переменная, в которой хранится информация о затратах и доходе рекламных кампаний по различным источникам.
+Необходимо дополнить исходную структуру показателем ROI, который рассчитаем по формуле: ((revenue / cost) - 1) * 100
+
+Пример работы программы:
+
+results = {
+    'vk': {'revenue': 103, 'cost': 98},
+    'yandex': {'revenue': 179, 'cost': 153},
+    'facebook': {'revenue': 103, 'cost': 110},
+    'adwords': {'revenue': 35, 'cost': 34},
+    'twitter': {'revenue': 11, 'cost': 24},
+}
+Результат:
+
+{'adwords': {'revenue': 35, 'cost': 34, 'ROI': 2.94},
+ 'facebook': {'revenue': 103, 'cost': 110, 'ROI': -6.36},
+ 'twitter': {'revenue': 11, 'cost': 24, 'ROI': -54.17},
+ 'vk': {'revenue': 103, 'cost': 98, 'ROI': 5.1},
+ 'yandex': {'revenue': 179, 'cost': 153, 'ROI': 16.99}}
+
+"""
+
+
+ROI_KEY = 'ROI'
+
+
+def ROI_calc(revenue, cost):
+    if cost == 0:
+        # Тут нужен exception и запись в лог. Но мы это пока не проходили, поэтому просто возвращаем 0
+        return 0
+
+    return round(((revenue / cost) - 1) * 100, 2)
+
+
+def ROI_add(to_add_roi):
+    for val in to_add_roi.values():
+        val[ROI_KEY] = ROI_calc(val['revenue'], val['cost'])
```

Comment:
As soon as we have ROI_KEY here, why shouldn't we have the same for 'revenue' and 'cost'? Let's make it consistent.